HB/builders.elpi

```diff
@@ -62,6 +100,7 @@ builders.end :- std.do! [
 /* ----------------------------- private code ------------------------------ */
 /* ------------------------------------------------------------------------- */
 
+pred factory.cdecl->w-mixins i:context-decl, o:w-mixins context-decl.
```

Comment:
@gares you can now try to make this line disappear without triggering a warning, good luck :sweat_smile:

Reply:
:magic_trick:

Reply:
After this: https://github.com/LPCIC/coq-elpi/issues/171 I'm not afraid of hearing voices anymore

Reply:
> After this: LPCIC/coq-elpi#171 I'm not afraid of hearing voices anymore

Aaaaah! I stumbled on this one at some point today and got stuck for 30min...